ui/packages/catalog-ui-search/src/main/webapp/component/query-editor/query-editor.view.js

```diff
@@ -165,9 +170,13 @@ module.exports = Marionette.LayoutView.extend({
     }
     queryContentView.save()
     this.queryTitle.currentView.save()
+    if (this.model.get('title') === '') {
+      this.setDefaultTitle()
+    }
     if (store.getCurrentQueries().get(this.model) === undefined) {
       store.getCurrentQueries().add(this.model)
     }
+    this.model.preQueryPlugin(store.getCurrentQueries())
```

Comment:
just curious, what does this line do?

Reply:
this will 'sanitize' the default title that is being set - in other words strip down the cql that is being used as the title to only the attributes that are being used in the query

Reply:
I'd prefer if we pulled out that `getSanitizedCql` method from there and set the title directly. Seems to me we are doing a lot of extra work calling that whole `preQueryPlugin`, plus that method is also setting other attributes and makes me worried that could cause another issue